deepchem/models/losses.py

```diff
@@ -1112,6 +1112,58 @@ def loss(pred_edge, inputs):
         return loss
 
 
+class DeepGraphInfomaxLoss(Loss):
```

Comment:
Can you add a dedicated unit test for this loss?

Reply:
done